src/local/delete.go

```diff
@@ -11,30 +11,25 @@ import (
 )
 
 // DeleteModel from manifest and disk
-func (m *Manager) DeleteModel(record manifest.Record) error {
+func (m *Manager) DeleteModel(record model.Record) error {
 	errors := utils.NewMultiError()
 
 	// Remove record from manifest content
 	m.manifest.DeleteRecord(record)
 
-	// Delete metadata file
-	if err := os.Remove(filepath.Join(m.ProjectDir(), record.MetaFilePath())); err == nil {
-		m.logger.Debugf("Removed \"%s\"", record.MetaFilePath())
-	} else if !os.IsNotExist(err) {
+	// Metadata file
+	if err := m.remove(m.Naming().MetaFilePath(record.RelativePath())); err != nil {
```

Comment:
`metaFilePath`, `configFilePath` sa generuju cez `naming` aby to bolo vsetko na jednom mieste.